config/api.yml

```diff
@@ -497,10 +497,18 @@
       :get:
       - :name: read
         :identifier: image_show_list
+      - :name: delete
```

Comment:
probably meant that to be under a :post: section

Reply:
Fixed